processing/stenosis_connector_test.go

```diff
@@ -57,7 +57,7 @@ import (
 // 	entry.JSONLine = string(jsonData)
 // }
 
-func TestStenosisQueryRegularSuccessForwarded(t *testing.T) {
+func _TestStenosisQueryRegularSuccessForwarded(t *testing.T, ifaceSetting string) []string {
```

Comment:
Just to understand, since this looks like a Pythonidiom: the leading underscore is to make sure it is not exported? Simply lowercasing is not good enough?

Reply:
Yes, lowercase would work as well, I guess. Just wanted to be more explicit...